src/main/g8/default.properties

```diff
@@ -1,7 +1,9 @@
-name=zio-akka-quickstart
+name=zio-scala2-quickstart
 description=This is a seed project which creates ready to use Scala application using ZIO, Akka HTTP and Slick.
```

Comment:
It's about time to update this description.

Reply:
done